fiftyone/operators/executor.py

```
@@ -828,30 +860,154 @@ def target_view(self, param_name="view_target", require_flat=False):
+
+        Returns:
+            a :class:`fiftyone.core.collections.SampleCollection`
+        """
+        return self._get_active_view(
+            sample_collection,
+            require_flat=True,
+            slices=slices,
+            media_type=media_type,
+        )
+
+    def _get_active_view(
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_get_active_view`

(ANN202)